docs/manual/comparison_with_sql.rst

```diff
@@ -0,0 +1,782 @@
+.. py:currentmodule:: datatable
+
+Comparison with SQL
+====================
+
+This page provides some examples of how various `SQL <https://en.wikipedia.org/wiki/SQL>`__ operations can be performed in ``datatable``. The ``datatable`` library is still growing; as such, not all functions in ``SQL`` can be replicated yet. If there is a feature you would love to have in ``datatable``, please make a feature request on the `github issues <https://github.com/h2oai/datatable/issues>`__ page.
+
+Most of the examples will be based on the famous `iris <https://en.wikipedia.org/wiki/Iris_flower_data_set>`__ dataset. `SQLite <https://www.sqlite.org/>`__ will be the flavour of SQL used in the comparison.
+
+Let's import ``datatable`` and read in the data:
+
+.. code-block:: python
+
+    from datatable import dt, f, g, by, join, sort, update, fread
+
+    file_path = '/datatable/docs/_static/iris.csv'
```

Comment:
I guess you can even do file_path = 'http://github.com/h2oai/datatable/docs/_static/iris.csv', then it will work even if there is no local `datatable` folder.